xgboost_synthetic/fairing_util.py

```diff
@@ -0,0 +1,20 @@
+import json
```

Comment:
Is this file intended to be checked in fairing? If so, can you merge it into fairing? If not, can you move it to `util.py`?

Reply:
I have deleted functions which are checked in fairing and the remaining one wound not. I will move it to `util.py`. Thanks!